README.template.md

```diff
@@ -0,0 +1,40 @@
+
+{{ ossHeader }}
+
+<h1 id="keyv-null-img-align-right-src-logo-svg-alt-keyv-logo-title-keyv-logo-width-100-">Keyv - Momento <img align="right" src="./keyv-logo.svg" alt="Keyv logo" title="Keyv logo" width="100"></h1>
+<p>This project is part of the <a href="https://www.npmjs.com/package/keyv">Keyv</a> suite.</p>
+
+## What and why?
+
+This project is a [Momento](https://gomomento.com) storage adaptor for [Keyv](https://github.com/jaredwray/keyv)
+
+TTL functionality is handled directly by momento so no timestamps are stored and expired keys are cleaned up internally.
+
+## Installation
+
+```shell
+npm install --save @gomomento-poc/node-keyv-adaptor
+```
+
+## Usage
+
+```ts
```

Comment:
FYI with the new v2 template engine you can just use an `include` tag to pull in as many code files as you want:

https://github.com/momentohq/client-sdk-javascript/blob/main/README.template.md?plain=1#L16

not a blocker for this pr but just letting you know, that way we can make sure ci runs the code

Reply:
Ill do in follow up PR good call!